components/scroll-wrapper/scroll-wrapper.js

```diff
@@ -53,11 +53,11 @@ class ScrollWrapper extends FocusVisiblePolyfillMixin(RtlMixin(LitElement)) {
 			:host([hidden]) {
 				display: none;
 			}
-
 			.d2l-scroll-wrapper-container {
 				box-sizing: border-box;
 				outline: none;
 				overflow-x: auto;
+				overflow-y: var(--d2l-scroll-wrapper-overflow-y, visible);
```

Comment:
I think you tried `auto` originally and then switched to `visible`.  Did you run into an issue with `auto`?

Reply:
Yes, auto failed the vdiff tests, it looked related as well since it was a failure associated with the wrapper. These failures now are associated with some unrelated component